content/docs/deployment/ghpages/contents.lr

```diff
@@ -45,14 +45,17 @@ with behavior for GitHub Pages.
 ## CNAME Support
 
 If you want to use a [CNAME :ext](https://en.wikipedia.org/wiki/CNAME) with
-GitHub pages and Lektor you can provide the intended CNAME with the `?cname`
-parameter:
+custom domain configured GitHub pages and Lektor you can provide the intended 
```

Comment:
I am not a native speaker, but this sentence does not sound grammatically correct to me.